cmd/thanos/tools.go

```diff
@@ -39,26 +41,33 @@ func checkRulesFiles(logger log.Logger, files *[]string) error {
 
 	for _, fn := range *files {
 		level.Info(logger).Log("msg", "checking", "filename", fn)
-		f, err := os.Open(fn)
+		matches, err := filepath.Glob(fn)
+		if matches == nil {
```

Comment:
We should check the `err` first and then `matches`, because `matches` will most likely be `nil` in case there is some other error and we will be overwriting that error with "Matching file not found".

Reply:
Right :thinking: